src/assets/styles/scss/components/landing/hero/_hero.scss

```diff
@@ -0,0 +1,10 @@
+@media all and (max-width: 650px) {
```

Comment:
Let's use the media query mixins in base.scss to keep responsiveness consistent.